src/schema/v2/sale_artwork.ts

```diff
@@ -92,7 +92,13 @@ export const SaleArtworkType = new GraphQLObjectType<any, ResolverContext>({
     return {
       ...SlugAndInternalIDFields,
       cached,
-      artwork: { type: Artwork.type, resolve: ({ artwork }) => artwork },
+      artwork: {
+        type: Artwork.type,
+        resolve: (root) => {
+          const { artwork } = root
+          return artwork
+        },
+      },
```

Comment:
This file probably doesn't need to change

Reply:
yep, cleaned up